src/uws_sys/libuwsockets.cpp

```
@@ -20,6 +20,30 @@ static inline std::string_view stringViewFromC(const char* message, size_t lengt
+        memcpy(data->remoteAddress, buf, len);
+        data->remoteAddressLength = (uint8_t)len;
+        data->remotePort = (uint16_t)p;
+      } else {
+        data->remoteAddressLength = 0xff;
+      }
+    }
+    if (data->remoteAddressLength == 0xff) return 0;
+    memcpy(out, data->remoteAddress, data->remoteAddressLength);
+    *port = data->remotePort;
+    return data->remoteAddressLength;
+  }
```

Cppcheck (2.21.0)

[style] The function 'layoutAssert' is never used.

(unusedFunction)